### PR TITLE
fix(tooltip) Fixed typo in tooltip.

### PR DESCRIPTION
**Pull request by ljelinkova**
The OK tooltip is sometimes 'Everyting is OK' and sometimes 'Everyting
is OK.'. This causes problems in E2E tests.

### CODE REVIEW

**Comment by jiekang** [comment]
@ljelinkova Could you rebase this branch on top of the latest upstream master branch? Thanks!

**Comment by joshuawilson** [comment]
@jiekang the original code has one with and one without the period. If we need someone to choose, we should ask UX @catrobson

**Comment by catrobson** [comment]
Without period.  This is meant to be a phrase, not a sentence, in a tooltip.

File: src/app/space/create/deployments/apps/deployment-card.component.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ const STAT_THRESHOLD = .6;
 })
 export class DeploymentCardComponent implements OnDestroy, OnInit {
 
+  private static readonly OK_TOOLTIP: string = 'Everything is ok.';
```
Comment (jiekang) [inline src/app/space/create/deployments/apps/deployment-card.component.ts]: Please remove the period at the end, thanks!

**Comment by joshuawilson** [comment]
[test]

**Comment by ljelinkova** [comment]
I've updated the test.

**Comment by jiekang** [comment]
@ljelinkova The commit message long comment has a typo:

```
The OK tooltip is sometimes 'Everyting is OK' and sometimes 'Everyting
is OK.'. This causes problems in E2E tests.
```

s/Everyting/Everything

You can rebase the commit with 'r' (reword) to fix it.

**Comment by ljelinkova** [comment]
Sorry for that - it should be OK now.

**Comment by joshuawilson** [comment]
[test]

**Comment by jiekang** [comment]
[test]

**Comment by jiekang** [comment]
[test]

**Pull request merged**